okhttp/src/commonJvmAndroid/kotlin/okhttp3/EventListener.kt

```diff
@@ -70,6 +71,30 @@ abstract class EventListener {
   open fun callStart(call: Call) {
   }
 
+  /**
+   * Invoked for calls that were not executed immediately because resources weren't available. The
+   * call will remain in the queue until resources are available.
+   *
+   * Use [Dispatcher.maxRequests] and [Dispatcher.maxRequestsPerHost] to configure how many calls
+   * OkHttp performs concurrently.
+   */
+  open fun dispatcherQueueStart(
```

Comment:
Are start and end the right terms for this?

dispatcherQueued and dispatcherExecuted?

Reply:
I'm thinking about it like spans in a tracing system or flame graph. Where did this call spend its first 800 milliseconds? In the dispatcher queue.